08_install-introduce-self-git.Rmd

```diff
@@ -14,6 +14,8 @@ substituting your name and **the email associated with your GitHub account**.
 
 From RStudio, go to *Tools > Shell* and tell Git your name and **GitHub email** by typing (use your own name and email):
 
+Note: If you are on Windows, make sure this has opened *Git Bash*, and not Windows Command Prompt! If it has not, restarting RStudio may fix this issue. Otherwise, you can open *Git Bash* from the Windows menu, or [change the option](https://support.rstudio.com/hc/en-us/articles/200532077-Version-Control-with-Git-and-SVN) via *Options -> Version Control*. 
```

Comment:
I believe in a Windows installation, when RStudio starts up it checks for Git Bash. If it's there, it will launch Git Bash from Tools > Shell. If not, it launches `cmd.exe`. So if Rstudio has not been restarted since Git Bash install, it will not know the git bash installation is present, and you can check this via Options > Version Control. 

Sorry, confirm which part?